src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -258,6 +258,12 @@
         Ref("NakedIdentifierFullSegment"),
     ),
     CascadeRestrictGrammar=OneOf("CASCADE", "RESTRICT"),
+    ExtendedTableReferenceGrammar=OneOf(
+        Ref("TableReferenceSegment"),
+        Sequence("ONLY", Ref("TableReferenceSegment")),
+        Sequence("ONLY", Bracketed(Ref("TableReferenceSegment"))),
```

Comment:
These 2 lines can be condensed to `Sequence("ONLY", OptionallyBracketed(Ref("TableReferenceSegment"))),`